src/main/js/controls/PairedResult/PairedResult.js

```diff
@@ -188,18 +189,49 @@ class PairedResult extends GraphContent {
      * @inheritDoc
      */
     resize(graph) {
+        let isPairedDataProper = true;
         if (utils.notEmpty(this.dataTarget.regions)) {
-            // If graph has more than 1 content then we dont load the regions
+            const regionList = this.dataTarget.regions;
+            const values = this.dataTarget.values;
+
+            //check if all region are there with respect to value
             if (graph.content.length > 1) {
-                hideAllRegions(graph.svg);
+                for (let i = 0; i < values.length; i++) {
+                    // eslint-disable-next-line max-depth
+                    for (const key in values[i]) {
+                        // eslint-disable-next-line max-depth
+                        if (!regionList.hasOwnProperty(key)) {
```

Comment:
Values is an array we can use array.every() on it but values[i] is an object we can not use array function on it.

Reply:
This can be further optimised to 
```js
const regionList = this.dataTarget.regions;
const values = this.dataTarget.values;
const isRegionMappedToAllValues = (value) =>
    Object.keys(value).every((v) =>
        regionList.hasOwnProperty(v)
    );
const isPairedDataProper = values.every(isRegionMappedToAllValues);
```

Reply:
ok i will optimise the code further